apps/ledger-live-desktop/src/renderer/hooks/useNftLinks.js

```diff
@@ -77,7 +83,13 @@ const linksPerCurrency = {
   ],
 };
 
-export default (account: Account, nft: ProtoNFT, metadata: NFTMetadata, onClose?: () => void) => {
+export default (
+  account: Account,
+  nft: ProtoNFT,
+  metadata: NFTMetadata,
+  onClose?: () => void,
+  isInsideDrawer?: boolean,
+) => {
```

Comment:
all of `useNftLink.js` is typed without flow syntax, i think you can safely turn it into a ts file

Reply:
![dd0-removebg-preview](https://user-images.githubusercontent.com/91890529/198328841-8b9784bb-2283-4d48-b931-12abc51752d3.jpg)